### PR TITLE
[INLONG-5460][Sort][Manager] Support Apache Hudi

### LINKED ISSUE #5099: [Umbrella] Support Apache Hudi
### Describe the proposal

Sort module supports Apache Hudi.

### Task list

- [ ] https://github.com/apache/inlong/issues/6646
- [ ] https://github.com/apache/inlong/issues/6647
- [ ] https://github.com/apache/inlong/issues/6648
- [ ] https://github.com/apache/inlong/issues/6649

### InLong Component

Other for not specified component

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Jellal-HT**
### Prepare a Pull Request

- Fixes #5099 

### Motivation

Support Apache Hudi in sort module and manager module

### Modifications

- Extend a new Extract Node for Apache Hudi
- Extend a new Load Node for Apache Hudi
- Add the corresponding flink connectors for Apache Hudi
- Extend Extract Node and Load Node in manager module for apache Hudi

### Verifying this change
- [x]  This change added tests and can be verified as follows:
  - add the Unit test of HudiLoadNode HudiLoadNodeTest
  - add the Unit test of HudiExtractNode HudiExtractNodeTest
  - add Hudi source service test HudiSourceServiceTest for manager module
  - add Hudi sink service test HudiSinkServiceTest for manager module

### CODE REVIEW

**Comment by XuQianJin-Stars** [comment]
hi @Jellal-HT A small suggested version upgrade to 0.12 for hudi.

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HudiLoadNode.java [hunk 1]
```diff
@@ -72,6 +79,7 @@ public Map<String, String> tableOptions() {
         Map<String, String> options = super.tableOptions();
         options.put("connector", "inlong-hudi");
```
Comment (gong) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HudiLoadNode.java:80]: Hi, connector is 'inlong-hudi', but I can't find code in sort-connector-hudi

**Comment by duowan1520** [comment]
Are you missing the step of bundling extract node and load node into the InLong sort ?

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HudiLoadNode.java (hunk 1 above)
Comment (ZuoFengZhang) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HudiLoadNode.java]: Thks, I have already dealt with it in another PR, will close this PR.

**Comment by dockerzhang** [comment]
duplicated with https://github.com/apache/inlong/pull/6789 and https://github.com/apache/inlong/pull/6790, close it.

**Pull request closed without merging**